addons/stock/static/tests/tours/stock_report_tests.js

```diff
@@ -15,6 +15,12 @@
     {
         trigger: '.btn[id="stock.view_diagram_button"]',
     },
+    {
+        trigger: '.btn-primary > span:contains("Overview")',
+    },
+    {
+        trigger: '.btn-primary > span:contains("Save")',
+    },
```

Comment:
????

Reply:
This tour uses the demo product "Acoustic Bloc Screens", which I've modified in this PR to make variants. 

So, this tour, click on the first demo product ("Acoustic Bloc Screens"), then go to the "Inventory" tab and click on `View Diagram`. As this is now a product variant, you need to choose which "View Diagram" variant you want to see, so I've added these 2 triggers, to select and save the first variant.